src/components/AssetMatching/components/ActionsToolbox/SecondaryActions.tsx

```diff
@@ -8,17 +8,22 @@ import ClickAwayListener from 'components/utils/ClickAwayListener';
 
 interface Props {
   secondaryActions: MatchingAction[];
+  actionButtonFill?: boolean;
```

Comment:
can we use our definition of naming bool? https://orfium.atlassian.net/wiki/spaces/ENG/pages/1335591044/General+Code+Rules